mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -71,11 +76,45 @@ export class MetricsPlotPanel extends React.Component {
       popoverX: 0,
       popoverY: 0,
       popoverRunItems: [],
+      timerId: null,
     };
     this.displayPopover = false;
+    this.intervalId = null;
     this.loadMetricHistory(this.props.runUuids, this.getUrlState().selectedMetricKeys);
   }
 
+  clearIntervalIfExists = () => {
+    if (this.intervalId) {
+      clearInterval(this.intervalId);
+    }
+  };
+
+  componentDidMount() {
+    if (!this.props.allRunsCompleted) {
+      this.intervalId = setInterval(() => {
+        const { runUuids } = this.props;
+        // Use `message` for demo purposes.
+        message.success({
+          content: 'Updated',
+          duration: 0.5,
+          style: {
+            marginTop: '20vh',
+          },
+        });
```

Comment:
@cedkoffeto I added this block for demo purposes, but do you think it's useful if the UI shows a message every time it updates the plot?

Reply:
@harupy  Not that useful in my point of view

I think we can show the status (live) somewhere maybe in a green box something like that to specify that there are executions not yet completed and only keep the completion message.
```python
message.success('All runs completed');
```

Reply:
> keep the completion message.

I agree!

> I think we can show the status (live) somewhere maybe in a green box something like that to specify that there are executions not yet completed and only keep the completion message.

This sounds good to me :) I'll take to the team about this.

Reply:
@harupy Ok  good 👍🏽